packages/palette/src/elements/Image/Image.shared.tsx

```diff
@@ -32,14 +32,18 @@ export interface BaseImageProps {
   style?: object
   /** Flag for if image should be lazy loaded */
   lazyLoad?: boolean
+  /** Event handler for right-clicking on the image */
+  onContextMenu?: (e: any) => void
 }
 
 export interface ImageProps
   extends BaseImageProps,
     SpaceProps,
     WidthProps,
     HeightProps,
-    BorderRadiusProps {}
+    BorderRadiusProps {
+  preventRightClick?: boolean
```

Comment:
This file represents the code that is _shared_ between Reaction and Emission. I don't think this is the right file to add these props in. 

I broke these down when working through the image lazy loading work and I didn't do a good enough job of documenting that fact. Sorry about that.